internal_workflow_testsuite.go

```diff
@@ -296,6 +364,26 @@ func (env *testWorkflowEnvironmentImpl) overrideActivity(activityFn, fakeActivit
 	env.overrodeActivities[fnName] = fakeActivityFn
 }
 
+func (env *testWorkflowEnvironmentImpl) overrideWorkflow(workflowFn, fakeWorkflowFn interface{}) {
+	// verify both functions are valid workflow func
```

Comment:
this is the same code for overrideActivity() - consider refactoring the validation into its own helper - something like validateFunctionOverride(actual, fake) string